Classes/Service/ProcessingService.php

```diff
@@ -155,7 +155,18 @@ public function getNodeFromRow(string $table, array $row, array $parentPointer =
             $usedElements[$table][$row['uid']]['count'] = 1;
         }
         $usedElements[$table][$row['uid']]['parentPointers'][] = $parentPointerString;
-        
+
+        if ($parentPointer['table'] == 'pages') {
```

Comment:
Please use ===

Reply:
fixed in 3c90734aac740a847013a542c218a595c698a6b8